.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           root: .
   html-proofer:
     executor:
-      name: hugo/cibuilds
+      name: hugo/default
```

Comment:
This name was changed in Orb. Keeping the old name causes a build error.

<img width="500" src="https://github.com/GSA/digitalgov.gov/assets/3385219/75b6e210-7107-491d-a1a6-14288fb3d529"/>

**Source:** https://app.circleci.com/pipelines/github/GSA/digitalgov.gov/18863